### ISSUE TITLE
Could not evaluate: Resource cTentacleAgent was not found.

### ISSUE THREAD

**Issue by dogjarek** [OctopusDeploy/OctopusDSC#186]
after an upgrade to the latest version last night (4.0.537) Puppet returns an error saying that `Could not evaluate: Resource cTentacleAgent was not found.`. Any ideas why?

```PS C:\Windows\system32> Get-DSCResource -Name cTentacleAgent

ImplementedAs   Name                      ModuleName                     Version    Properties
-------------   ----                      ----------                     -------    ----------
PowerShell      cTentacleAgent            OctopusDSC                     4.0.522    {Name, ApiKey, CommunicationMode, CustomPublicH...
PowerShell      cTentacleAgent            OctopusDSC                     4.0.537    {Name, ApiKey, CommunicationMode, CustomPublicH...```

**Comment by stopthatastronaut** [comment]
That's pretty interesting. I can't see any changes in the tentacle part of the code in teh last couple of upgrades - which version did you update from?

**Comment by dogjarek** [comment]
From the one which is on the list `4.0.522` -> `4.0.537`

**Comment by stopthatastronaut** [comment]
Very strange. That's the most recent change, which maps to pull #185. We didn't touch the tentacle resource or the module manifests themselves (though we did make a small change to a shared file).

I had a quick google around and I got this from Puppet, suggesting that having two separate DSC Module versions installed side by side will throw a very similar looking error

https://tickets.puppetlabs.com/browse/MODULES-2818 

That's a very old ticket, but from your pasted output above, it does kinda look like you have two side-by-side versions. Removing the older one might (fingers crossed) get you past the issue?
